docs/api-specificatie/ac/openapi.yml

```diff
@@ -0,0 +1,1193 @@
+openapi: 3.0.0
```

Comment:
heel de map `docs/api-specificatie` mag niet meegecommit worden - deze worden op build-time gekopieerd

Reply:
Ik neem hem mee met de gitignore. Die map is lokaal nodig voor een lokale versie van de documentatie site.

Reply:
Klopt, die wordt ook in een shell-script zo klaargezet, maar die werken enkel op Linux natuurlijk :-)

Reply:
opgelost